elk/extraction/extraction.py

```diff
@@ -192,11 +191,9 @@ def extract_hiddens(
         template_path=cfg.template_path,
         rank=rank,
         world_size=world_size,
-        seed=cfg.seed,
```

Comment:
This seed is used in load_prompts, why was it removed?

Reply:
@AlexTMallen what do you mean, it is still there or not?